docs/source/index.rst

```diff
@@ -0,0 +1,53 @@
+Welcome to Modalities's documentation!
```

Comment:
Shouldn't it be "Modalities' documentation"? I don't want to be overly critical, but this is on the first page in the first line ... ;-)

Reply:
Changed :)